docs/learning.rst

```diff
@@ -7,37 +7,59 @@ Learning The Stack
+=================== =========== =================
+Component           Technology  Style/Methodology
+=================== =========== =================
+Database            PostgreSQL  Django ORM       
+Backend Framework   Django      pycodestyle/Black            
+Frontend Framework  React/Redux ESLint/Prettier 
+CSS Framework       SCSS        BEM/Airbnb       
+=================== =========== =================
 
 Tutorials and Resources
 ~~~~~~~~~~~~~~~~~~~~~~~
 
 Learning the Backend
 ####################
 
-**Django** is a Python Web framework that provides a huge number of tools for web developers to quickly write scalable code with minimal configuration. It is used all over the tech industry by companies like Spotify, Instagram, YouTube, and DropBox!
+**Django** is a Python Web framework that provides a huge number of tools for web
+developers to quickly write scalable code with minimal configuration. It is used all
+over the tech industry by companies like Spotify, Instagram, YouTube, and DropBox!
+
+`Writing your first Django app
+<https://docs.djangoproject.com/en/3.2/intro/tutorial01/>`_ is the official Django
+tutorial. It is top notch! The official documentation can be found at the same url and
+provides high quality information about how to build with this modern web framework. For
+example, here's the documentation on `making queries with Django
+<https://docs.djangoproject.com/en/3.2/topics/db/queries/>`_ 
 
-`Writing your first Django app <https://docs.djangoproject.com/en/2.2/intro/tutorial01/>`_ is the official Django tutorial. It is top notch! The official documentation can be found at the same url and provides high quality information about how to build with this modern web framework.
 
 Learning React/Redux
 ####################
 
-React is a Javascript library created by Facebook for "building user interfaces". It allows developers to make encapsulated components that can be written once and used anywhere. 
+React is a Javascript library created by Facebook for building user interfaces. It
+allows developers to make encapsulated components that can be written once and used
+anywhere. 
 
 Redux is state container that makes React development easier to manage long term! 
 
-If you're a beginner, `we've created a React tutorial <https://github.com/noahpresler/React-Tutorial>`_ that will teach you the basics of developing a React app all via codepen. This is a great way to get started! 
+The official docs are the go-to: `React Basics
+<https://reactjs.org/docs/hello-world.html>`_, `React Hooks
+<https://reactjs.org/docs/hooks-intro.html>`_, `Redux & Redux Toolkit
+<https://redux.js.org/tutorials/index>`_, and `TypeScript
+<https://www.typescriptlang.org/docs/>`_. We suggest going through the step-by-step
+React tutorial rather than the practical tutorial because the practical tutorial is done
+with class-based components rather than functional components, which we are trying to
```

Comment:
Remove "which we are trying to shift towards." We can assume they'll notice these changes, and it'll be done fully in the near future.

Reply:
We can't assume this. If 90% of our codebase is class based React, "what's the harm in adding another one?"
"We can just convert it later."
This expectation needs to be clear from the start.

Reply:
Alright but please add a commented note in the file. I think there's another similar "we're trying to do it now" line elsewhere. The same applies to anything else that'll be removed or changed eventually.

Reply:
I'm not sure what the point is. If they're already looking at the .rst file, they were probably looking to update it anyways. I've removed it in favor of "preferring functional components."

Reply:
The documentation hasn't been updated in some time, it's helpful for other (and future) developers to do some quick fixes/updates if they know where they are. The hope is that they won't have to read through everything to find something unusual.